### PR TITLE
degiro updates

### PR DESCRIPTION
**Pull request by Joren-vanGoethem**
added 'sell' detection
and make GetQuantity and GetUnitPrice currency agnostic and buy/sell agnostic

### CODE REVIEW

**Comment by VibeNL** [comment]
Thanks for the contribution. I will have a closer look tomorrow, but could you at least add a testcase for the sell activity?

**Comment by Joren-vanGoethem** [comment]
will do, btw add some labels for PR's like feature/bug/enhancement/... ;)

**Comment by Joren-vanGoethem** [comment]
i added a few unit tests, i think the 'example' naming you used for directories of csv files is quite useless to be honest
'example' doesn't tell you anything about the files in that directory
i added very small and simple unit tests that only test one simple thing like buy euro older, sell euro order, ...

File: GhostfolioSidekick/FileImporter/DeGiro/DeGiroParser.cs [hunk 1]
```diff
@@ -146,17 +150,24 @@ protected override CsvConfiguration GetConfig()
...
+        {
+            // oop is the same for both buy and sell or Koop and Verkoop in dutch
+            // dont include currency at the end, this can be other things than EUR
+            var quantity = Regex.Match(record.Omschrijving, $"oop (?<amount>\\d+) @ (?<price>[0-9]+,[0-9]+)").Groups[1].Value;
+
 			return decimal.Parse(quantity, GetCultureForParsingNumbers());
 		}
 
-		private decimal GetUnitPrice(DeGiroRecord record)
+		private decimal GetUnitPrice(DeGiroRecord record, bool buy = true)
```
Comment (VibeNL) [inline GhostfolioSidekick/FileImporter/DeGiro/DeGiroParser.cs:161]: not sure why there is an unused parameter 👼

**Review by VibeNL — APPROVED** [review]
Just a single unused parameter, nothing major :) . Looks good!

**Comment by VibeNL** [comment]
> i added a few unit tests, i think the 'example' naming you used for directories of csv files is quite useless to be honest 'example' doesn't tell you anything about the files in that directory i added very small and simple unit tests that only test one simple thing like buy euro order, sell euro order, ...

IT is a work in progress, I like the suggestion 👍 . BTW, the build fails, likely on a culture problem

File: GhostfolioSidekick/FileImporter/DeGiro/DeGiroParser.cs (hunk 1 above)
Comment (Joren-vanGoethem) [reply GhostfolioSidekick/FileImporter/DeGiro/DeGiroParser.cs]: you'll have to approve the new changes

**Comment by VibeNL** [comment]
I will merge it and fix it in master. Need this code for another change :)

**Pull request merged**